manifests/v1beta1/components/controller/controller.yaml

```diff
@@ -40,6 +40,17 @@ spec:
             - containerPort: 8080
               name: metrics
               protocol: TCP
+            - containerPort: 18080
+              name: healthz
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: healthz
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: healthz
```

Comment:
Should we also add [startupProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes)?

Reply:
Sure, with startup probe, we could setup more graceful period only when startup.

I didn't investigate deeply what values are the appropriate for `failureThreshold`, `periodSeconds`, `initialDelaySeconds` and other values, I just use the [default values](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) now.

But if we have some standards to determine readiness, liveness and startup, we will need to seperate those values in detail.

Do you think we need to test to find more appropriate values for each ? @tenzen-y

Reply:
Sorry for my late response. @anencore94 
It might be better to set each parameter referred to [default values](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) since I think that it is hard to optimize these parameters.

Reply:
I also agree with you. Since currently there is no clue to specification

Reply:
Also referenced at #1973